pkg/e2etemplates/apiextensions_chart_config_e2e_chart_values.go

```diff
@@ -0,0 +1,17 @@
+package e2etemplates
+
+const ApiextensionsChartConfigE2EChartValues = `chart:
```

Comment:
Is there a reason for these names?

Reply:
I followed the naming from AWS:
https://github.com/giantswarm/e2etemplates/blob/a53b3bf982c863948719acb122321ee511dce813/pkg/e2etemplates/apiextensions_aws_config_e2e_chart_values.go#L3
It could be improved but I am unsure what would be better.

Reply:
https://github.com/giantswarm/apiextensions/tree/master/helm/apiextensions-chart-config-e2e-chart

- `apiextensions-` - we need that to avoid conflicts between repos.
- `chart-config-e2e` - arbitrary name.
- `-chart` - we need that to not conflict with docker image names. This is unfortunate but this is how it is in quay.